.github/CHANGELOG.md

```diff
@@ -2,6 +2,58 @@
 
 <h3>New features since last release</h3>
 
+* Added the gate `PauliRot(angle, pauli_word)` that performs an arbitrary 
+  Pauli rotation specified by the Pauli word in string form and the gate
+  `MultiRZ(angle)` that performs a rotation generated by a tensor product
+  of Pauli Z operators.
+  [(#559)](https://github.com/XanaduAI/pennylane/pull/559)
+  
+  Consider as an example the following circuit:
+
+  ```python
+  dev = qml.device('default.qubit', wires=4)
+
+  @qml.qnode(dev)
+  def circuit(angle):
+      qml.PauliRot(angle, "IXYZ", wires=[0, 1, 2, 3])
+      return [qml.expval(qml.PauliZ(wire)) for wire in [0, 1, 2, 3]]
+  ```
+
+  We can run this circuit and look at the corresponding circuit drawing
+  
+  ```python
+  >>> circuit(0.4)
+  [1.         0.92106099 0.92106099 1.        ]
+  >>> print(circuit.draw())
+   0: ──╭RIXYZ(0.4)──┤ ⟨Z⟩
```

Comment:
Just a question here: would starting the name of the gate with `RP` to match the formula in the docs make sense? (no strong preferences here, just came to my mind)

Reply:
Actually, what do you think about
```
   0: ──╭RI(0.4)──┤ ⟨Z⟩
   1: ──├RX(0.4)──┤ ⟨Z⟩
   2: ──├RY(0.4)──┤ ⟨Z⟩
   3: ──╰RZ(0.4)──┤ ⟨Z⟩
```
This is also better because you could have `PauliRot(0.4, 'XYZ', wires=[3,2,1])` in which case `RXYZ` would be confusing because spread out among the wires you would actually do `RZYX`.

Reply:
Yes, I think this looks nicer!